CODIGO/modNetwork.bas

```diff
@@ -18,19 +18,48 @@ Attribute VB_Name = "modNetwork"
 Option Explicit
 
 Private Client As Network.Client
+
+Private Type t_FailedIp
+    IP As String
+    Port As String
+End Type
+
+Dim FailedIpList(10) As t_FailedIp
+Public FaiedListSize As Integer
+
 #If PYMMO = 0 Then
 Public Function IsConnected() As Boolean
     IsConnected = Connected
 End Function
 #End If
 
+Public Function IsFailedString(ByVal IP As String, ByVal Port As String)
+    Dim i As Integer
+    For i = 0 To UBound(FailedIpList) - 1
+        If FailedIpList(i).IP = IP And FailedIpList(i).Port = Port Then
+            IsFailedString = True
+            Exit Function
+        End If
+    Next i
+End Function
+
+Public Function AddFailedIp(ByVal IP As String, ByVal Port As String) As Boolean
+    FailedIpList(FaiedListSize).IP = IP
+    FailedIpList(FaiedListSize).Port = Port
+    FaiedListSize = FaiedListSize + 1
+    AddFailedIp = True
+    If FaiedListSize = UBound(servers_login_connections) Then
```

Comment:
For clarity and readability, I would

Move this out of the function 
`If FaiedListSize = UBound(servers_login_connections) Then
        AddFailedIp = False
    End If`
    
to `RetryWithAnotherIp` and turn `AddFailedIp` into a `sub` not a `function`